jitter/styles.css

```diff
@@ -0,0 +1,75 @@
+body {
+  margin: 0;
+  font-family: 'Space Mono', monospace;
+}
+canvas {
+  display: block;
+  vertical-align: bottom;
+}
+
+#tsparticles {
```

Comment:
Since `fullScreen` options are enabled, this CSS is not necessary.

Reply:
Understood. I’ll make the changes in a bit.